apps/web/lib/checkForEmptyAssignment.ts

```diff
@@ -0,0 +1,28 @@
+// This function checks if EventType requires assignment.
+// returns true: if EventType requires assignment but there is no assignment yet done by the user.
+// returns false: for all other scenarios.
+export function checkForEmptyAssignment({
+  assignedUsers,
+  hosts,
+  isManagedEventType,
+  assignAllTeamMembers,
+  isTeamEvent,
+}: {
+  assignedUsers: any;
+  hosts: any;
```

Comment:
This is not good practice and would be better to avoid it. Instead, you can import the types `HOST` and `ChildrenEventType` from the main event-type component or declare them here.

Reply:
updated